src/zarr_benchmarks/parse_json_for_plots.py

```diff
@@ -76,161 +79,334 @@ def get_benchmarks_dataframe(
 def plot_relplot_benchmarks(
     data: pd.DataFrame,
     *,
-    group: str,
     x_axis: str,
     y_axis: str,
-    hue: str,
-    size: str,
-    title: str,
-    output_filename: str | None = None,
+    sub_dir: str,
+    output_filename: str,
+    title: str | None = None,
+    hue: str | None = None,
+    size: str | None = None,
+    col: str | None = None,
```

Comment:
Can you document (just short one line comments is fine, or a proper docstring if you prefer) what the parameters are? At least for `col` it's not obvious what it is for.